src/static/menu-actions.json

```diff
@@ -233,6 +233,10 @@
         "EDIT_VECTOR_MASK": {
             "$action": "layers.editVectorMask",
             "$enable-rule": "one-layer-selected-has-vector-mask"
+        },
+        "ADD_VECTOR_MASK": {
+            "$action": "layers.addVectorMask",
+            "$enable-rule": "one-layer-selected-can-add-vector-mask"
```

Comment:
Per comment above, these two should be a combination of `layers-selected-1` and some other simpler rule